BookAndCanvas/SQL/setupDB.sql

```diff
@@ -0,0 +1,132 @@
+create database BookAndCanvas
+
+create table Users
+(
+	[Id] int primary key identity(1,1),
+	[FName] nvarchar(50) not null,
+	[LName] nvarchar(50) not null,
+	[Phone] nvarchar(50) not null,
+	[Email] nvarchar(50) not null,
+	[ImgUrl] nvarchar(50),
+	[Bio] nvarchar (max)
+)
+
+create table UserMsg
+(
+	[Id] int primary key identity(1,1),
+	[MessageId] int not null,
+	[BuyerId] int not null,
+	[SellerId] int not null
+)
+
+alter table UserMsg
+add constraint FK_UserMsg_User_Buyer 
+	foreign key (BuyerId)
+	references Users (Id)
+
+alter table UserMsg
+add constraint FK_UserMsg_User_Seller 
+	foreign key (SellerId)
+	references Users (Id)
+
+create table Message
+(
+	[Id] int primary key identity(1,1),
+	[MessageId] int not null,
+	[MsgDate] DateTime not null,
```

Comment:
msgDate - I think we should use a default date method.  `DEFAULT GETDATE() `? This way we don't have to manually insert a date every time a record gets created.

Reply:
Again, I do not understand how this method is appliued when I create the database, can you please clarify?